dev-java/icu4j/icu4j-69.1.ebuild

```diff
@@ -0,0 +1,153 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom pom.xml --download-uri https://github.com/unicode-org/icu/archive/refs/tags/release-69-1.tar.gz --slot 69 --keywords "~amd64 ~arm64 ~ppc64 ~x86" --ebuild icu4j-69.1.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="com.ibm.icu:icu4j:69.1"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="A set of Java libraries providing Unicode and Globalization support"
+HOMEPAGE="https://icu-project.org/"
+SRC_URI="https://github.com/unicode-org/icu/archive/refs/tags/release-${PV/./-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="icu"
+SLOT="69"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	>=virtual/jdk-1.8:*
+	test? (
+		dev-java/junitparams:0
+	)
+"
+
+RDEPEND="
+	>=virtual/jre-1.8:*
+"
+
+BDEPEND="
+	app-arch/zip
+"
+
+S="${WORKDIR}/icu-release-${PV/./-}/icu4j/main/"
+
+JAVA_SRC_DIR="classes"
+JAVA_GENTOO_CLASSPATH_EXTRA="icu4j.jar:icu4j-charset.jar:icu4j-localespi.jar"
+
+JAVA_TEST_SRC_DIR=(
+	"tests"
+	"../tools/misc/src/com/ibm/icu/dev/tool/locale"
+)
+
+JAVA_TEST_RESOURCE_DIRS=(
+	"tests"
+	"tests/core/src/com/ibm/icu/dev/data"
+)
```

Comment:
for the purpose of the ebuild, you should split test resources and test java classes as is done for maven projects. `java-pkg-simple` doesn't work correctly when sources and resources are mixed together.